r10k.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'log4r',     '1.1.10'
   s.add_dependency 'multi_json', '~> 1.10'
 
-  s.add_dependency 'puppet_forge', '~> 2.2.8'
+  s.add_dependency 'puppet_forge', '~> 2.3.0'
```

Comment:
Would it be worth setting this to `'~> 2.3'` so we don't need to bump when there's a Y release?

Reply:
I wondered... there's definitely a mix of pin levels in here. I went with this because it matches what we did in master, but I could go either way.

Reply:
I'm all for consistency, so we can just leave this for now.